tests/snapshots/TestCLICommands_atmos_atlantis_--help.stdout.golden

```diff
@@ -0,0 +1,44 @@
+
+
+Usage:
+
+  atmos atlantis [sub-command] [flags]
+
+
+Available Commands:
+
+  generate                       Generate Atlantis configuration files
+
+
+Flags:
+
+    -h, --help    help for atlantis (default "false")
+
+
+Global Flags:
+
+        --logs-file string          The file to write Atmos logs to. Logs can be
+                                    written to any file or any standard file
+                                    descriptor, including '/dev/stdout',
+                                    '/dev/stderr' and '/dev/null' (default
+                                    "/dev/stdout")
+
+        --logs-level string         Logs level. Supported log levels are Trace,
+                                    Debug, Info, Warning, Off. If the log level
+                                    is set to Off, Atmos will not log any
+                                    messages (default "Info")
+
+        --redirect-stderr string    File descriptor to redirect 'stderr' to.
+                                    Errors can be redirected to any file or any
+                                    standard file descriptor (including
+                                    '/dev/null'): atmos <command>
+                                    --redirect-stderr /dev/stdout
+
+The '--' (double-dash) can be used to signify the end of Atmos-specific options 
```

Comment:
Old formatting. Double dash should be grouped with flags

Reply:
I still have to refactor tests. Also a lot of things are getting slightly updated. So this is not exactly updated one.